x509/install.sh

```diff
@@ -1,17 +1,10 @@
-#!/bin/bash
-
-set -eu -o pipefail
-shopt -s failglob
-
-cp -a $(go env GOROOT)/src/crypto/x509/* ./
-rm ./x509_test.go
-
-# The following code is utterly horrifying.  I can't find a better way to do it.  :(
-mkdir -p ${GOPATH}/src/vendor/golang.org/
-cp -R $(go env GOROOT)/src/vendor/golang_org/* ${GOPATH}/src/vendor/golang.org/
-# h/t to https://www.cyberciti.biz/faq/unix-linux-replace-string-words-in-many-files/
-OLD_PACKAGE='"golang_org/x/crypto/cryptobyte'
-OLD_PACKAGE="${OLD_PACKAGE//\//\\/}"
-NEW_PACKAGE='"golang.org/x/crypto/cryptobyte'
-NEW_PACKAGE="${NEW_PACKAGE//\//\\/}"
-sed -i "s/${OLD_PACKAGE}/${NEW_PACKAGE}/g" ./*.go ${GOPATH}/src/vendor/golang.org/x/crypto/cryptobyte/*.go
+#!/bin/sh
+set -e
+gopath="$(go env GOROOT)"
+x509path=$gopath/src/crypto/x509
+test -d $gopath
+test -d $x509path
+cp -av $x509path/testdata .
+cp -v $x509path/*.* .
+rm x509_test.go
+sed -i.bak 's/golang_org/golang.org/g' x509.go
```

Comment:
It looks like you've removed the copying of the `cryptobyte` package... is that copy actually unnecessary now?  I seem to remember it being required when I was porting to Go 1.10.x, is that no longer the case?

Reply:
no longer necessary it seems